### ISSUE TITLE
rose suite-run: Jinja2 filters leaking into ssh command

### ISSUE THREAD

**Issue by oliver-sanders** [metomi/rose#2077]
If a Jinja2 filter writes to stdout this somehow leaks into the ssh command during `rose suite-run`. Writing to stderr has no effect.

For instance the custom Jinja2Filter:
```python
def filter_name(_):
    print 'hello world'
```
When used in a suite results in the traceback:
```
[FAIL] ssh -oBatchMode=yes -oStrictHostKeyChecking=no -oConnectTimeout=8 -n world@...
```

**Comment by matthewrmshin** [comment]
(This may get fixed by #2132 - as STDIN will then be redirected from `/dev/null` by default.)

**Comment by matthewrmshin** [comment]
Closed by #2132.
